api/document/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from document.views import TreeView
+
+urlpatterns = [
+    url(r'^(?P<policy_id>\d+)(/(?P<identifier>[a-zA-Z0-9_-]+))?',
```

Comment:
We might want to use `[0-9]` instead of `\d` here, as in Python 3 `\d` matches anything flagged in Unicode as a numeral digit, such as `൬`. Not a big deal, but if we're expecting policy numbers to only be 0-9 we might as well go with the more restrictive option.

Reply:
Good to know and can do. I wonder how well Django/psycopg/etc. handle that...